README.md

```diff
@@ -102,28 +118,29 @@ and CML set up on an Ubuntu LTS base with CUDA libraries and
 
 ### CML Functions
 
-CML provides a number of helper functions to help package the outputs of ML
-workflows (including numeric data and visualizations about model performance)
-into a CML report.
+CML provides a number of functions to help package the outputs of ML workflows
+(including numeric data and visualizations about model performance) into a CML
+report.
 
 Below is a table of CML functions for writing markdown reports and delivering
 those reports to your CI system (GitHub Actions or GitLab CI).
 
-| Function                | Description                                                    | Inputs                                                      |
-| ----------------------- | -------------------------------------------------------------- | ----------------------------------------------------------- |
-| `cml-runner`            | Starts a runner locally or in cloud providers                  | See [Arguments](https://github.com/iterative/cml#arguments) |
-| `cml-publish`           | Publish an image for writing to CML report.                    | `<path to image> --title <image title> --md`                |
-| `cml-send-comment`      | Return CML report as a comment in your GitHub/GitLab workflow. | `<path to report> --head-sha <sha>`                         |
-| `cml-send-github-check` | Return CML report as a check in GitHub                         | `<path to report> --head-sha <sha>`                         |
-| `cml-pr`                | Create a pull request.                                         | TODO                                                        |
-| `cml-tensorboard-dev`   | Return a link to a Tensorboard.dev page                        | `--logdir <path to logs> --title <experiment title> --md`   |
+| Function                | Description                                                      | Example Inputs                                              |
+| ----------------------- | ---------------------------------------------------------------- | ----------------------------------------------------------- |
+| `cml-runner`            | Launch a runner locally or hosted by a cloud provider            | See [Arguments](https://github.com/iterative/cml#arguments) |
+| `cml-publish`           | Publicly host an image for displaying in a CML report            | `<path to image> --title <image title> --md`                |
```

Comment:
I assume this is indeed "publicly."

Reply:
Actually, `cml-publish --native` on GitLab should protect assets behing the same access control as the project, I think. 

When using our asset hosting service, there is no protection: if you have access to our S3 bucket or know the hash of the file, you can access it.

Obviously, if you know the hash of the file it would (probably) mean that you already have acces to the file, and guessing 128 bits over a network connection would be like sweeping the Great Wall of China with a nail polish brush.

Reply:
I should have checked the documentation before even assuming this kind of behavior. 🙃  GitLab doesn't require authentication for image files, and does for every other kind of file. Given that our main use case involves image files, you're absolutely right.

<a href="https://docs.gitlab.com/ee/security/user_file_uploads.html"><i><blockquote>
<p>Images that are attached to issues, merge requests, or comments do not require authentication to be viewed if they are accessed directly by URL. This direct URL contains a random 32-character ID that prevents unauthorized people from guessing the URL for an image, thus there is some protection if an image contains sensitive information.</p>
<p>Authentication is not enabled because images must be visible in the body of notification emails, which are often read from email clients that are not authenticated with GitLab, such as Outlook, Apple Mail, or the Mail app on your mobile device.</p>
<p><b>Non-image attachments do require authentication to be viewed.</b></p></blockquote></i></a>

Reply:
So yes, "publicly" is a pretty safe assumption to make.